gemfiles/rails_5.0.gemfile

```diff
@@ -3,5 +3,6 @@
 source "https://rubygems.org"
 
 gem "rails", "5.0.0"
+gem "tzinfo-data"
```

Comment:
This also needs to be added to the `gemfiles/rails_4.2.gemfile`

Reply:
This file is autogenerated